completion/git-machete.completion.zsh

```diff
@@ -65,7 +65,12 @@ _git-machete() {
           _arguments '1:: :__git_machete_directions_go' && ret=0
           ;;
         (github)
-          _arguments '1:: :__git_machete_github_subcommands' && ret=0
+          _arguments \
```

Comment:
I'd add some `case` over the subcommand here if easily doable, so that the matching options are only prompted to `create-pr` or `checkout-prs` (as for Bash)